vm/Cargo.toml

```diff
@@ -77,6 +77,7 @@ generational-arena = "0.2"
 parking_lot = "0.11"
 thread_local = "1.0"
 cfg-if = "0.1.10"
+memchr = "2.1.0"
```

Comment:
It seems `bstr::ByteSlice::find` is compatible with memchr.  In that case, because we already uses bstr, I think we don't need to add this crate

https://docs.rs/bstr/0.2.13/bstr/trait.ByteSlice.html#method.find